### ISSUE TITLE
building from source doesn't complete correctly

### ISSUE THREAD

**Issue by chris-ramon** [sourcegraph/srclib#281]
While following the `srclib` installation steps documented on [https://srclib.org/install/](https://srclib.org/install/), found that installation from source won't complete correctly.

*Consider to the next gist as a temp workaround: https://gist.github.com/chris-ramon/7a5c779862fa94151c79ab2075c40fa7* 

**Steps to reproduce:**

1.  `go get -u -v sourcegraph.com/sourcegraph/srclib/cmd/srclib`
2. `srclib toolchain install go`

**Actual:**
Step 2 does not complete correctly:

```bash
Makefile:31: recipe for target 'govendor' failed
make: *** [govendor] Error 2
failed to install/upgrade Go (sourcegraph.com/sourcegraph/srclib-go) toolchain: command ["make"] failed: exit status 2
```

See more: https://gist.github.com/chris-ramon/dd37614031dae01a2bbc9273daff4515

**Expected:**
Step 2 completes without errors.

**Env:**
```bash
(srclib-go)-> go version
go version go1.6 linux/amd64
```

```bash
(srclib-go)-> echo $GOPATH
/home/chris/go_packages
```

```bash
(srclib-go)-> pwd && git log -n 1 --oneline
/home/chris/go_packages/src/sourcegraph.com/sourcegraph/srclib-go
6831ef7 Remove fetch of golang.org/x/tools/go/types
```

```bash
(srclib)-> pwd && git log -n 1 --oneline
/home/chris/go_packages/src/sourcegraph.com/sourcegraph/srclib
b7f0f2e Merge pull request #280 from sourcegraph/fix-create-version
```

**Comment by beyang** [comment]
Thanks for filing.

**Comment by alexsaveliev** [comment]
My bad, sorry. Was broken by #268, should be fixed by #282. Please test now

**Comment by chris-ramon** [comment]
Thanks for addressing it @alexsaveliev, confirmed locally that #282 fixes the issue.
